package.json

```diff
@@ -25,6 +25,7 @@
     "react-native-image-picker": "jp928/react-native-image-picker#6ee35b69f3dbd6c7c66f580fd4d9eabf398703d4",
     "react-native-invertible-scroll-view": "1.0.0",
     "react-native-keyboard-aware-scroll-view": "0.2.8",
+    "react-native-keyboard-input": "^3.0.22",
```

Comment:
can we make this an exact version you can accomplish that with `yarn add -E react-native-keyboard-input`